### PR TITLE
[YouTube] | added HomeTask2.6/ Youtube task

### PR DESCRIPTION
**Pull request by bommas12**
@SaiTeja2904 Please review the hometask2.6

### CODE REVIEW

File: YouTube/scripts/addVideoElement.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const template = document.querySelector("#list-item");
+
+function addVideoElements(totalWidth, videoArray, itemsPerPage) {
+
+  const fragment = document.createDocumentFragment();
+  videoArray.forEach((videoItem) => {
+    const { thumbnail, author, title, description, publishTime, views, videoId } = videoItem;
```
Comment (SaiTeja2904) [inline YouTube/scripts/addVideoElement.js]: Please extract to a separate function.

File: YouTube/scripts/eventListeners.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  const statisticsList = await getStatistics(videosIdList);
+  state.thumbnailData = getThumbnailData(videosList, statisticsList);
+
+  const thumnailSlider = document.querySelector(".thumbnail-slider");
+  thumnailSlider.innerHTML = "";
+  thumnailSlider.appendChild(addVideoElements(thumnailSlider.clientWidth, state.thumbnailData, breakPoints.active.items));
+  thumnailSlider.style.transition = "transform 0.4s ease-in-out";
+  thumnailSlider.style.transform = "translateX(0px)";
+
+  const pagContainer = document.querySelector(".pagination-container");
```
Comment (SaiTeja2904) [inline YouTube/scripts/eventListeners.js]: Make it as pageContainer instead of pagContainer

File: YouTube/scripts/paginate.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+async function pag(event) {
```
Comment (SaiTeja2904) [inline YouTube/scripts/paginate.js]: What is pag? Could you please explain or rename.

File: YouTube/scripts/services.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+function Services(api, { statisticsURL, searchURL, nextURL }) {
```
Comment (SaiTeja2904) [inline YouTube/scripts/services.js]: Please change the function name "Services" to another name which follows camelCase Notation

**Review by SaiTeja2904 — CHANGES_REQUESTED** [review]

File: YouTube/scripts/services.js (hunk 4 above)
Comment (bommas12) [reply YouTube/scripts/services.js]: named it as services

File: YouTube/scripts/paginate.js (hunk 3 above)
Comment (bommas12) [reply YouTube/scripts/paginate.js]: named it as paginate

File: YouTube/scripts/addVideoElement.js (hunk 1 above)
Comment (bommas12) [reply YouTube/scripts/addVideoElement.js]: extracted in separate function createVideoElement

**Review by SaiTeja2904 — APPROVED** [review]

**Pull request merged**